build_tools/travis/install.sh

```diff
@@ -78,6 +78,11 @@ elif [[ "$DISTRIB" == "ubuntu" ]]; then
     source testvenv/bin/activate
     pip install pytest pytest-cov cython==$CYTHON_VERSION
 
+elif [[ "$DISTRIB" == "32bit" ]]; then
+    # At the time of writing numpy 1.9.1 is included in the travis
+    # virtualenv but we want to use the numpy installed through apt-get
+    docker run --rm -v`pwd`:/io quay.io/pypa/manylinux1_i686 /bin/bash -c "set -e; export PATH=/opt/python/cp36-cp36m/bin:$PATH; pip install numpy scipy cython nose pytest; cd /io; pip install -e l;  pytest -l sklearn"
```

Comment:
Maybe add a script with the commands under `build_tools/travis/install_docker_32bit.sh` ?